config/config.yml

```diff
@@ -39,3 +39,14 @@ ldap:
   guess_email:
     enabled: false
     attr: ""
+
+# Set first_user_admin to true if you want that the first user that signs up
+# becomes the admin.
```

Comment:
`Set first_user_admin to true if you want that the first user that signs up to be an admin.`